app/src/main/java/com/example/todoapp/view/adapter/TaskAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.todoapp.view.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.todoapp.R
+import com.example.todoapp.databinding.TodoItemBinding
+import com.example.todoapp.model.Task
+
+class TaskAdapter(
+    var list: ArrayList<Task>
+): RecyclerView.Adapter<TaskAdapter.TaskViewHolder>() {
+
+    class TaskViewHolder(view: View): RecyclerView.ViewHolder(view) {
+        val binding = TodoItemBinding.bind(view)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) = TaskViewHolder(
+        LayoutInflater.from(parent.context).inflate(R.layout.todo_item, parent, false)
+    )
+
+    override fun onBindViewHolder(holder: TaskViewHolder, position: Int) {
+        val item = list[position]
+        with(holder) {
```

Comment:
with(holder) ko có ý nghĩa
em nên viết phần bind data này ở holder
```suggestion
        holder.bindData(item) 
```